docs/about/join.md

```diff
@@ -41,24 +45,33 @@ To apply for an account on the Wynton HPC cluster, please fill out the following
  
  <div class="form-group">
   <label for="email_address">UCSF email address</label> <span style="color: red">*</span><br>
-  <input type="email" id="email" name="email" autocomplete="email" placeholder="Enter email" size="30" required><br>
+  <input type="email" id="email" name="email" autocomplete="email" placeholder="Enter email" size="30" required><br> 
+  <small>(**External collaborators only: Please provide your preferred email address)</small>
  </div>
  
- <div class="form-group">
-  <label>UCSF affiliation</label> <span style="color: red">*</span><br>
+ <div class="form-group"> 
+  <label for="affiliation">UCSF affiliation</label> <span style="color: red">*</span><br>
   <fieldset>
-   <input type="radio" id="affiliation_0" name="affiliation" value="faculty" required/> Faculty<br>
-   <input type="radio" id="affiliation_1" name="affiliation" value="postdoc"/> Postdoc<br>
-   <input type="radio" id="affiliation_2" name="affiliation" value="staff"/> Staff, research assistant, etc.<br>
-   <input type="radio" id="affiliation_3" name="affiliation" value="student_lab"/> Student (part of a lab)<br>
-   <input type="radio" id="affiliation_4" name="affiliation" value="student_rotating"/> Student (rotating student)<br>
-   <input type="radio" id="affiliation_5" name="affiliation" value="student_misc"/> Student (miscellaneous)<br>
-   <input type="radio" id="affiliation_6" name="affiliation" value="sysadm"/> Sysadm<br>
-   <input type="radio" id="affiliation_7" name="affiliation" value="other"/> Other
-   <input type="text" id="affiliation_other" name="affiliation_other" placeholder="Collaborating with, ..." minlength="2" size="30"><br>
+   <input type="radio" id="affiliation_0" name="affiliation" value="Faculty"> Faculty<br>     
+   <input type="radio" id="affiliation_1" name="affiliation" value="Postdoc"> Postdoc<br>
+   <input type="radio" id="affiliation_2" name="affiliation" value="Staff"> Staff<br>
+   <input type="radio" id="affiliation_3" name="affiliation" value="Research_Assistant"> Research Assistant<br>
+   <input type="radio" id="affiliation_4" name="affiliation" value="Student"> Student<br>
+   <input type="radio" id="affiliation_5" name="affiliation" value="other"> Other<br>
   </fieldset>
  </div>
-
+ <div class="form-group">
+  <label for="other_desc">If 'Other', please clarify </label><br>
+  <input type="text" id="other_desc" name="other_desc" placeholder="Collaborating with, ..." size="30" minlength="2"><br>
+ </div>
+ 
+ <div class="form-group">
+  <label for="student_affiliation"> If 'Student', please select appropriate option</label> <br>
```

Comment:
Same here, there's no HTML tag with an `id = "student_affiliation"`.  Is this correct?

Reply:
Updated code and validated